controls/SV-244549.rb

```diff
@@ -15,46 +15,36 @@
 can be accomplished by physical means (e.g., employing physical distribution
 systems) or by logical means (e.g., employing cryptographic techniques). If
 physical means of protection are employed, then logical means (cryptography) do
-not have to be employed, and vice versa.
+not have to be employed, and vice versa.'
+  desc 'check', 'Verify SSH is installed with the following command:
 
+$ sudo yum list installed openssh-server
 
-  "
-  desc  'rationale', ''
-  desc  'check', "
-    Verify SSH is installed with the following command:
+openssh-server.x86_64                 8.0p1-5.el8          @anaconda
 
-    $ sudo yum list installed openssh-server
+If the "SSH server" package is not installed, this is a finding.'
+  desc 'fix', 'Install SSH packages onto the host with the following command:
 
-    openssh-server.x86_64                 8.0p1-5.el8          @anaconda
-
-    If the \"SSH server\" package is not installed, this is a finding.
-  "
-  desc  'fix', "
-    Install SSH packages onto the host with the following command:
-
-    $ sudo yum install openssh-server.x86_64
-  "
+$ sudo yum install openssh-server.x86_64'
   impact 0.5
   tag severity: 'medium'
   tag gtitle: 'SRG-OS-000423-GPOS-00187'
-  tag satisfies: ['SRG-OS-000423-GPOS-00187', 'SRG-OS-000424-GPOS-00188',
-'SRG-OS-000425-GPOS-00189', 'SRG-OS-000426-GPOS-00190']
+  tag satisfies: ['SRG-OS-000423-GPOS-00187', 'SRG-OS-000424-GPOS-00188', 'SRG-OS-000425-GPOS-00189', 'SRG-OS-000426-GPOS-00190']
   tag gid: 'V-244549'
-  tag rid: 'SV-244549r743896_rule'
+  tag rid: 'SV-244549r916422_rule'
   tag stig_id: 'RHEL-08-040159'
   tag fix_id: 'F-47781r743895_fix'
   tag cci: ['CCI-002418']
   tag nist: ['SC-8']
 
   if virtualization.system.eql?('docker')
     impact 0.0
-    describe "Control not applicable within a container" do
-      skip "Control not applicable within a container"
+    describe 'Control not applicable within a container' do
+      skip 'Control not applicable within a container'
     end
   else
     describe package('openssh-server') do
       it { should be_installed }
     end
   end
```

Comment:
This should really be amended to only be NA if we are in a container and openssh-server is not installed.